frc/src/main/scala/com/lynbrookrobotics/potassium/frc/WPIClock.scala

```diff
@@ -21,8 +21,14 @@ object WPIClock extends Clock {
         lastTime = Some(currentTime)
         running = false
       }
+    }
+
+    val notifier = new Notifier(() => {
+      onTick()
     })
 
+    onTick()
```

Comment:
Is there a a reason why we're `onTick()` this before we schedule?

Reply:
Nope, fixed!